modules/nw-udn-additional-config-details.adoc

```diff
@@ -8,26 +8,29 @@
 
 The following table explains additional configurations for `ClusterUserDefinedNetwork` and `UserDefinedNetwork` custom resources (CRs) that are optional. It is not recommended to set these fields without explicit need and understanding of OVN-Kubernetes network topology.
 
-.`UserDefinedNetworks` optional configurations
-[cols="1,1,7", options="header"]
+. Optional configurations for user-defined networks
+[cols="2,1,7", options="header"]
 |====
 
-|Field|Type|Description
+|CUDN field|UDN field|Type|Description
 
-|`spec.joinSubnets`
+|`spec.network.<topology>.joinSubnets`
+|`spec.<topology>.joinSubnets`
 |object
 |When omitted, the platform sets default values for the `joinSubnets` field of `100.65.0.0/16` for IPv4 and  `fd99::/64` for IPv6. If the default address values are used anywhere in the cluster's network you must override it by setting the `joinSubnets` field. If you choose to set this field, ensure it does not conflict with other subnets in the cluster such as the cluster subnet, the `default` network cluster subnet, and the masquerade subnet.
 
 The `joinSubnets` field configures the routing between different segments within a user-defined network. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
 
-|`spec.ipam.lifecycle`
+|`spec.network.<topology>.ipam.lifecycle`
+|`spec.<topology>.ipam.lifecycle`
 |object
-|The `spec.ipam.lifecycle` field configures the IP address management system (IPAM). You might use this field for virtual workloads to ensure persistent IP addresses. The only allowed value is `Persistent`, which 
-ensures that your virtual workloads have persistent IP addresses across reboots and migration. These are assigned by the container network interface (CNI) and used by OVN-Kubernetes to program pod IP addresses. You must not change this for pod annotations. 
+|The `spec.ipam.lifecycle` field configures the IP address management system (IPAM). You might use this field for virtual workloads to ensure persistent IP addresses. The only allowed value is `Persistent`, which
+ensures that your virtual workloads have persistent IP addresses across reboots and migration. These are assigned by the container network interface (CNI) and used by OVN-Kubernetes to program pod IP addresses. You must not change this for pod annotations.
 
-Setting a value of `Persistent` is only supported when `spec.ipam.mode` is set to `Enabled`. 
+Setting a value of `Persistent` is only supported when `spec.ipam.mode` is set to `Enabled`.
```

Comment:
.spec.network.layer2.ipam.mode

Reply:
I would avoid spesifying the topology name in this context, how about: "ipam.mode is set to .."

Reply:
@ormergi so are you saying we should change this to "Setting a value of `Persistent` is only supported when `spec.network.<topology>.ipam.mode` is set to `Enabled`"?